prefect_census/client.py

```diff
@@ -0,0 +1,94 @@
+"""Module containing client for interacting with the Census API"""
+from httpx import AsyncClient, Response
+
+
+class CensusClient:
+    """
+    Client for interacting with the Census API.
+
+    Args:
+        api_key: API key to authenticate with the Census API.
+    """
+
+    def __init__(self, api_key: str):
+        self._closed = False
+        self._started = False
+
+        self.client = AsyncClient(
+            base_url="https://app.getcensus.com",
+            headers={"Authorization": f"Bearer {api_key}"},
```

Comment:
Do we need user-agent here like [dbt](https://github.com/PrefectHQ/prefect-dbt/blob/main/prefect_dbt/cloud/clients.py#L27) or does Census not allow it?

The goal of user-agent is for accounting how many Prefect users use the Census collection I think.

Reply:
For sure!  I wasn't sure why that was there. Added.